frontend/src/app/libs/common-components/components/input/input.component.ts

```diff
@@ -47,6 +47,8 @@ export class InputComponent implements ControlValueAccessor, OnInit, OnDestroy,
 
 	@Input() public label: string;
 
+	@Input() public size: string;
```

Comment:
can we set width of input outside by just using css, seems like this solution might be overkill for simple width setting :)